plutus-core/plutus-core/src/PlutusCore/Bitwise/Convert.hs

```diff
@@ -3,7 +3,7 @@
 {-# LANGUAGE OverloadedStrings #-}
 
 -- | Implementations for conversion primops from 'Integer' to 'ByteString' and back again.
-module PlutusCore.Builtin.Convert (
+module PlutusCore.Bitwise.Convert (
```

Comment:
Thanks! `Builtin` is for general builtins matter, not for anything Cardano-specific, so this is a helpful change.